internal/clients/sources/sources_client.go

```diff
@@ -164,7 +164,7 @@ func (c *SourcesClient) GetProvisioningTypeId(ctx context.Context) (string, erro
 
 func (c *SourcesClient) loadAppId(ctx context.Context) (string, error) {
 	ctxval.Logger(ctx).Info().Msg("Fetching the Application Type ID of Provisioning for Sources")
-	resp, err := c.client.ListApplicationTypes(ctx, &ListApplicationTypesParams{})
+	resp, err := c.client.ListApplicationTypes(ctx, &ListApplicationTypesParams{}, headers.AddSourcesIdentityHeader)
```

Comment:
Oh this is devel use only tho, right, because ListApplicationTypes is actually not gated by authentication IMHO, or is it? It should be publicly available endpoint

Reply:
According to sources docs, looks like this endpoint is gated by the authorization header/identity 
https://redocly.github.io/redoc/?url=https://github.com/RedHatInsights/sources-api-go/raw/main/public/openapi-3-v3.1.json#tag/application-types/operation/listApplicationTypes

Reply:
Interesting, that it works locally :) but what do I know. Lets add it and hope it won't become an issue later on :)

Reply:
Maybe 3scale sets it to some default value that causes it to fail on stage. Weird.